README.md

```diff
@@ -1,10 +1,12 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 # Bookstore CMS
 
```

Comment:
[OPTIONAL] It's preferable not to leave the acknowledgment section blank so that your README is properly documented. (You might thank the design provider or programmers who assisted you in finishing the project)

Reply:
Thanks, @medaminedev66 for suggesting these changes. It's solved.